File: src/main/java/com/woomulwoomul/woomulwoomulbackend/api/service/develop/DevelopService.java
package com.woomulwoomul.woomulwoomulbackend.api.service.develop;

import lombok.RequiredArgsConstructor;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

@Service
@Transactional(readOnly = true)
@RequiredArgsConstructor
public class DevelopService {

    @Value("${api.name}")
    private String serverName;

    /**
     * 서버명 조회
     * @retrun 서버명
     */
    public String getServerName() {
        return serverName;
    }
}

File: admin-api/src/main/kotlin/com/woomulwoomul/adminapi/service/develop/DevelopService.kt
package com.woomulwoomul.adminapi.service.develop

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Service
import org.springframework.transaction.annotation.Transactional

@Service
@Transactional(readOnly = true)
class DevelopService(
    @Value("\${api.name}") private val serverName: String,
) {

    /**
     * 서버명 조회
     * @return 서버명
     */
    fun getServerName(): String {
        return serverName
    }
}